test/TestUserAcceptance.hs

```diff
@@ -898,3 +917,15 @@ buildAnsiRegex attrs fgs bgs =
     choice l r = "(" <> l <> "|" <> r <> ")"
   in
     choice tmux24 tmux25
+
+-- | Interleaves stderr with stdout
+-- Does not accept STDIN arguments, since we do not need to communicate with the
+-- process
+readProcessWithErrorOutput_ :: ProcessConfig stdinClosed stdout stderr -> IO T.Text
+readProcessWithErrorOutput_ pc = do
+  (_, out) <- withProcess_ config $ \p -> atomically $ (,) <$> waitExitCodeSTM p <*> getStdout p
+  pure $ T.decodeUtf8 $ LB.toStrict out
```

Comment:
Danger, Will Robinson!

This is non-total.  If there are ANSI escape codes etc, won't this fail (when the return `T.Text` value is demanded)?
Shouldn't we return a `ByteString` and let the caller decide what to do with it?